### ISSUE TITLE
Incorrect duplicate flagging because of using different computers

### ISSUE THREAD

**Issue by tonicebrian** [org-roam/org-roam#1162]
### Description
I use a Mac at work and a Ubuntu at home. I sync my org-roam files between computers using Google Drive. But from time to time I get these messages at startup:

```
A: /home/cebrian/google-drive/OrgRoam/how_to_export_ebook_highlights_to_your_digital_notes_youtube.org
B: /Users/cebriant/Google Drive/OrgRoam/how_to_export_ebook_highlights_to_your_digital_notes_youtube.org

skipping...
```

#### Steps to Reproduce

I don't really know how to reproduce but I guess it is if I leave the Emacs app open in my Mac and then go to the Linux and edit that same file.

1. Have two copies in different computers of your org roam files
2. Sync them with an app (Dropbox, Google Drive, ...)
3. Work on them 
4. Next time you open Emacs from either computer you'll see the problem above.
...
-->

#### Expected Results

Org roam files' path is relative to some org-roam folder instead of storing full paths in the DB. 

#### Actual Results

A: /home/cebrian/google-drive/OrgRoam/how_to_export_ebook_highlights_to_your_digital_notes_youtube.org
B: /Users/cebriant/Google Drive/OrgRoam/how_to_export_ebook_highlights_to_your_digital_notes_youtube.org

### Environment

- Emacs: GNU Emacs 26.3 (build 2, x86_64-pc-linux-gnu, GTK+ Version 3.22.30)
 of 2019-09-16
- Framework: Doom
- Org: Org mode version 9.4 (nil @ /home/cebrian/.emacs.d/.local/straight/build/org-mode/)
- Org-roam: 1.2.0

**Comment by jethrokuan** [comment]
Yeah I was thinking of changing the default to a local directory. It won't really break anything: those who set `org-roam-db-location` will be unaffected, while those who did not change the defaults will just be doing a cache rebuild.

**Comment by tonicebrian** [comment]
I haven't much experience with Emacs Lisp but I'm proficient with Haskell and Scala. Let me know if this thing would be something I could contribute with some guidance from your side.

**Comment by jethrokuan** [comment]
This is resolved in #1164
